mbed_targets/mbed_targets.py

```diff
@@ -2,14 +2,15 @@
 
 mbed targets supports an online and offline mode, which instructs targets where to look up the target database.
 
-The entry points to the API are:
-- `get_target_by_product_code` function, which looks up an MbedTarget from its product code
-- `get_target_by_online_id` function, which looks up an MbedTarget by its slug and type
-The lookup can be from either the online or offline database, depending on the given mode.
-The mode can be one of the following DatabaseMode enum fields:
-    AUTO: the offline database is searched first, if the target isn't found the online database is searched.
-    ONLINE: the online database is always used.
-    OFFLINE: the offline database is always used.
+An MbedTarget can be retrieved by calling one of the public functions. All of which return an instance
+of MbedTarget.
+
+Functions:
```

Comment:
This is unnecessary as pdoc will generate a list of public members of the module.

Reply:
I'd be worried that this won't be maintained and will get out of date.

Reply:
Agreed, deleting this as pdoc should auto generate something useful anyway.

Reply:
I merged the master docstring changes and removed the functions/classes lists. We still list the entry points. Thoughts?

Reply:
If those entry points are public functions listed by pdoc it's not necessary to leave them there. It'll become an unnecessary maintenance burden.

Reply:
Yes, I was just looking at the pdoc artifacts and it makes no sense to list them. Removing in the next commit.

Reply:
ec5e0a0